api/src/test/java/marquez/db/SourceDaoTest.java

```diff
@@ -35,73 +41,133 @@
 
 @Category({DataAccessTests.class, IntegrationTests.class})
 public class SourceDaoTest {
+  private static final UUID ROW_UUID = UUID.randomUUID();
+  private static final Instant NOW = Instant.now();
+  private static final SourceType TYPE = newDbSourceType();
+  private static final SourceName NAME = newSourceName();
+  private static final URI CONNECTION_URL = newConnectionUrlFor(TYPE);
+  private static final String DESCRIPTION = newDescription();
```

Comment:
This can all be stored in a SourceRow so its typed and easier to identify usage of Source models